app/assets/styles/bulletin.css

```diff
@@ -10,46 +10,76 @@ figure {
 	margin: 0;
 }
 
-.primary-nav--banner {
-	width: 100%;
-	height: 2.5em;
-	background: rgb(167, 169, 172);
-	color: #fff;
-	margin-right: 1em;
-	display: flex;
-	justify-content: space-around;
-	border-bottom: 4px solid #ff9900;
+.home-hero {
+  height: 250px;
+  font-family: 'Trajan Pro Bold', 'Times New Roman', serif;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
```

Comment:
I feel like these values are shared just based on what the designs look like, but the best way to check is to make sure this branch has the latest changes that you merged into dev.